tests/suites/config.js

```diff
@@ -1,39 +1,40 @@
-module.exports = [{
+module.exports = [
+{
 	deviceType: process.env.DEVICE_TYPE,
-	suite: `${__dirname}/../suites/os`,
+	suite: `${__dirname}/../suites/hup`,
 	config: {
 		networkWired: false,
 		networkWireless: process.env.WORKER_TYPE === 'qemu' ? false : true,
+		downloadVersion: 'latest',
 		balenaApiKey: process.env.BALENACLOUD_API_KEY,
 		balenaApiUrl: 'balena-cloud.com',
 		organization: process.env.BALENACLOUD_ORG
 	},
-	image: `${__dirname}/balena.img.gz`,
+	image: `${__dirname}/balena-image.docker`,
 	workers: process.env.WORKER_TYPE === 'qemu' ? ['http://worker'] : {
 		balenaApplication: process.env.BALENACLOUD_APP_NAME,
 		apiKey: process.env.BALENACLOUD_API_KEY,
 	},
 },
 {
 	deviceType: process.env.DEVICE_TYPE,
-	suite: `${__dirname}/../suites/hup`,
+	suite: `${__dirname}/../suites/cloud`,
 	config: {
 		networkWired: false,
 		networkWireless: process.env.WORKER_TYPE === 'qemu' ? false : true,
-		downloadVersion: 'latest',
 		balenaApiKey: process.env.BALENACLOUD_API_KEY,
 		balenaApiUrl: 'balena-cloud.com',
 		organization: process.env.BALENACLOUD_ORG
 	},
-	image: `${__dirname}/balena-image.docker`,
+	image: `${__dirname}/balena.img.gz`,
 	workers: process.env.WORKER_TYPE === 'qemu' ? ['http://worker'] : {
```

Comment:
This is probably a separate logic change.

Reply:
I didn't change any logic in this file, just the order of the suites, git is having trouble with the diff.